provider/pkg/provider/provider.go

```diff
@@ -672,13 +672,15 @@ func (k *kubeProvider) Configure(_ context.Context, req *pulumirpc.ConfigureRequ
 		kubeClientSettings.QPS = &asFloat
 	}
 
+	// check all three locations config could be here and return error?
```

Comment:
You probably want to remove this comment?

Reply:
yes, thank you!